pandas/core/frame.py

```diff
@@ -7594,6 +7748,9 @@ def nlargest(
         descending order. The columns that are not specified are returned as
         well, but not used for ordering.
 
+        This method is equivalent to
+        ``df.nsorted(n, columns, ascending=False)``.
```

Comment:
I'd prefer if this line was removed; including `nsorted` in the `See Also` section below seems sufficient.